src/js/services/lang.js

```diff
@@ -78,40 +80,30 @@ const languageRequest = () => {
       }
     });
   });
-};
-
-const submitLanguage = () => {
-  const submitBtn = $('#submit-btn');
-  const input = $('#lang-req')[0];
 
-  if (input.value.length > 0) {
-    // If user input present, adjust UI and add event listener to submit data on click to amplitude
-    submitBtn.css({ opacity: '100%', cursor: 'pointer' });
-    submitBtn.on('click', () => {
-      clearInterval(checkLangInput);
-      langForm.css('pointer-events', 'none');
+  // Toggle submit button UI and button state based on user input
+  langForm.on('keyup', () => {
+    submitBtn.removeAttr('disabled');
```

Comment:
i think this logic could be simpler. `if (input.value == '') submitBtn.attr('disabled', 'disabled') else ...removeAttr(... els`

Reply:
^ Good point

Reply:
good call